dime-research-standards/pillar-3-research-reproducibility/checklists/tex/reproducibility-package-guidance-note.tex

```diff
@@ -0,0 +1,114 @@
+\input{checklist-header.tex}
+
+% ------------------------------ End of preamble ---------------------------------------------
+\begin{document}
+	\begin{fullwidth}
+
+\titleBox{
+	\textcolor{white}{\LARGE{\textbf{DIME Analytics \\ Reproducibility Check Guidance Note}} \\
+	\Large\textbf{{v2.0 - Last updated November 16, 2022}}}
+}
+
+	\section*{Reproducibility Check Guidance Note}
+
+The reproducibility check verifies that the code package as submitted:
+
+	\begin{enumerate}
+		\setlength\itemsep{-0.1em}
+		\item Is stable. It produces identical outputs each time it is run
+		\item Produces outputs that exactly match the analytical outputs (tables and data visualizations) in the final manuscript, without manual adjustments
```

Comment:
there's some gray area here though, no? My understanding is it's ok (though not encouraged) if they do some manual formatting as long as the results are the same (like we wouldn't want them to change the order of columns manually but changing the name or appearance of column headers would be ok)

Reply:
changed this line from
>Produces outputs that exactly match the analytical outputs (tables and data visualizations) in the final manuscript, without manual adjustments

to
> Produces outputs that match the analytical outputs (tables and data visualizations) in the final manuscript, without manual edits to numbers

in 3f063ff